recipes-bsp/u-boot/u-boot-ostree-scr/qemuriscv64/boot.cmd

```diff
@@ -4,6 +4,12 @@ then
     exit
 fi
 
+# Recent kernel commit in 5.3:
+# https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/arch/riscv/kernel?h=v5.3&id=671f9a3e2e24cdeb2d2856abee7422f093e23e29
+# changed page table setup which now crushes the qemu loaded DTB.
+# Let's copy the DTB to the established load addr instead where it's safe.
+cp ${fdt_addr} ${fdt_addr_r} 1000
```

Comment:
Ugh.  I actually don't like this being here.  This file won't get updated from version to version.  I just tried an update from 621 -> 627 and it didn't work because this file stays outside of the rootfs and isn't managed by OTA.

Reply:
Nm, since this is only for QEMU riscv targets atm, this is probably fine.  Noting this in a JIRA.